### PR TITLE
test: use gnfd-cmd to do e2e test

### PR DESCRIPTION
**Pull request by sysvm**
### Description

Use gnfd-cmd to do e2e test.

### Rationale

N/A

### Example

N/A

### Changes

N/A

### CODE REVIEW

**Review by actioncli — APPROVED** [review]

**Review by joeylichang — APPROVED** [review]

File: .github/workflows/e2e-test.yml [hunk 1]
```diff
@@ -44,49 +44,59 @@ jobs:
...
+          touch config.toml
+          {
+            echo rpcAddr = \"http://localhost:26750\"
+            echo chainId = \"greenfield_9000-121\"
+          } > config.toml
+          ./gnfd-cmd -c ./config.toml --home ./ sp ls
+          sleep 5
+          ./gnfd-cmd -c ./config.toml --home ./ bucket create gnfd://spe2etestbucket
+          sleep 5
+          ./gnfd-cmd -c ./config.toml --home ./ object put --contentType "application/json" $GITHUB_WORKSPACE/test/e2e/spworkflow/testdata/example.json gnfd://spe2etestbucket/example.json
```
Comment (will-2012) [inline .github/workflows/e2e-test.yml:98]: add an object payload > 16MB case?

**Review by will-2012 — APPROVED** [review]

File: .github/workflows/e2e-test.yml (hunk 1 above)
Comment (sysvm) [reply .github/workflows/e2e-test.yml:98]: ok

File: test/e2e/spworkflow/e2e_test.sh [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+      exit 1
+  fi
+}
+
+#######################
+# run sp workflow e2e #
+#######################
+function run_e2e() {
+  set -e
+  test_create_bucket
```
Comment (will-2012) [inline test/e2e/spworkflow/e2e_test.sh:158]: No need to check the return value?

File: .github/workflows/e2e-test.yml [hunk 3]
```diff
@@ -16,15 +16,7 @@ on:
       - fix-release*
 
 env:
-  GreenfieldTag: v0.2.1-alpha.1
-  GreenfieldIntegrationTestBranch: sp-refactor-0.2.1-new-sdk
-  GOPRIVATE: github.com/bnb-chain
   GH_ACCESS_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
```
Comment (will-2012) [inline .github/workflows/e2e-test.yml:19]: Remove all `GH_ACCESS_TOKEN ` in action yml.

File: .github/workflows/e2e-test.yml [hunk 4]
```diff
@@ -44,49 +36,23 @@ jobs:
...
         run: |
-          cd greenfield
-          bash ./deployment/localup/localup.sh all 1 7
-          bash ./deployment/localup/localup.sh export_sps 1 7 > sp.json
-          cd ..
-      # Run Greenfield-SPs
-      - name: Run Greenfield-SPs
+          bash ./test/e2e/spworkflow/e2e_test.sh --startSP
+      # Build Greefeld Cmd
+      - name: Build Greefeld Cmd
```
Comment (will-2012) [inline .github/workflows/e2e-test.yml:52]: `Greefeld `  typo.

File: .github/workflows/e2e-test.yml (hunk 3 above)
Comment (sysvm) [reply .github/workflows/e2e-test.yml:19]: fixed

File: test/e2e/spworkflow/e2e_test.sh (hunk 2 above)
Comment (sysvm) [reply test/e2e/spworkflow/e2e_test.sh:158]: I head bucket, download data and compare md5 to check data is correct.

Comment (sysvm) [reply test/e2e/spworkflow/e2e_test.sh:159]: If there is an error, action will fail.

**Pull request merged**